native/cocos/2d/renderer/RenderDrawInfo.h

```diff
@@ -256,7 +262,9 @@ class RenderDrawInfo final {
     struct DrawInfoAttrs {
         RenderDrawInfoType _drawInfoType{RenderDrawInfoType::COMP};
         bool _vertDirty{false};
-        bool _isMeshBuffer{false};
+        uint8_t _isMeshBuffer: 1;
+        uint8_t _isVertexPositionInWorld: 1;
+        uint8_t _padding: 6;
```

Comment:
Don't add a new boolean property, we could reuse the `byte` and make 8 bits for 8 boolean values.